src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.Android.cs

```diff
@@ -26,13 +26,22 @@ public void SetFrame(Rectangle frame)
 			if (Context == null)
 				return;
 
+
 			var left = Context.ToPixels(frame.Left);
 			var top = Context.ToPixels(frame.Top);
 			var bottom = Context.ToPixels(frame.Bottom);
 			var right = Context.ToPixels(frame.Right);
 			var width = Context.ToPixels(frame.Width);
 			var height = Context.ToPixels(frame.Height);
 
+			if (nativeView.Bottom == bottom &&
```

Comment:
Without this check the call to layout that used the same parameters caused a StackOverflowException

Reply:
@hartez  FYI To this code I had to do to make this PR work. Even we don't merge this PR this seems relevant